### DISCUSSION TITLE
Question on Burea sampleset and it’s files

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#973] [General]
Testing the tool wavpack I discovered, trying to pack Burea Church sampleset released by Lars Palo if I am correct, that it was already converted. In this case wavpack refuses to pack files. 
I had a look and disxoveees that all samples actually already come with .wv extension that means they are already processed with wavpack. 
ok. I have two questions ?
GO is capable to natively recognise .wv files ? It looks so. 
is it best for GO to provide wavpacked files with .wv extension or it doesn’t matter ? I mean : should packed files been left with .wav ext or they MUST be renamed with .wv ? I only wish a confirmation because I have already seen that Go read .wv files. Just to know the best way to do. Thanks.

**Comment by larspalo** [comment]
It doesn't matter to GO if the wavpacked samples have a .wv or .wav extension.

**Comment by ChasFW** [comment]
Paolo2504, It looks as though we are following similar trajectories in exploring GO's capabilities. You are asking the same questions a day or two before me! I had also been wondering as to the purpose and necessity of wavpack. I was wondering why Lars Palo's sound files are "wav", not wavpacked as ".wv", thinking that the sound files **_had_** to be wavpacked. But it seems not. Is this due to GO's evolution?

I have now successfully created the graphics (.organ file) for our local church's 10-stop (1M+P) organ. I did this with OrganBuilderPortable.exe to create the basics, followed by much tweaking to the file to get it right. Next, I need to take some samples from the organ itself. Lars Palo's excellent tutorial seems to be a good starting point, but is this tutorial still fully applicable to GO 3.5? Audacity seems to be an essential tool here. I understand the need for sound reduction, and Audacity can do this, but I don't understand the bit about looping. My Norton AV complains about Nick Appleton's AutoLoop, and won't allow the executable anywhere near my computer. It seems that Audacity can also handle looping, but it seems that all this does is to extend a short sample seamlessly into a longer one. Listening to Lars Palo's samples, it would seem that all that is required is a sample of a note being played for 3 seconds or so. I am probably missing something important here, but not sure what.

**Reply by paolo2504** [reply]
I didn’t know Organ builderpoetable: I’ll give a try. 
I know audacity but I don’t use it since time: I’ll use your hint to take time in seeing what’s new in recent release. 
anyway, I’m only interested in using Go, not in building organs (at least for now).
These days, I was much interested in wavpacking all my samplesets and convert them in packages. I wrote a batch script to do that. 
finally, Lars replied that GO is able to read .wav and .wv

**Reply by ChasFW** [reply]
https://virtualpipeorgans.wordpress.com/tools/organ-builder-by-vincent-forman/
